# degrees
float64 true_track_made_good

# degrees
bool magnetic_track_made_good_valid
float64 magnetic_track_made_good

float64 ground_speed_knots
float64 ground_speed_kph
